Police-Data-Accessibility-Project/data-sources-app
# fix: client tests

Will take care of #132 when merged to `main`.

To test:
`cd client`
`npm run coverage`

Ensure successful test run and that the coverage report prints to the console.
